gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #200: Don't clear filters when events page refreshes

### Description

* Used JavaScript to preserve the filters on the events page even after the page is refreshed
* Increased the test coverage to 98%

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1. Go to the events page
1. Choose some filters and apply them. See that the filtered events show up, as desired, and the filters are still shown, as desired.
1. Click "Interested" on some event and see that the filters are still shown, as desired.
![keep filters](https://user-images.githubusercontent.com/13837978/69926398-4c804480-1482-11ea-8799-505b9467bb9f.gif)


### Checklist
